.github/workflows/nix.yml

```diff
@@ -1,16 +0,0 @@
-name: "Nix"
```

Comment:
Even if it is currently brittle, we should not remove this CI test

Reply:
Added the nix.yml back